nixos/tests/all-tests.nix

```diff
@@ -686,6 +686,8 @@ in {
   oddjobd = handleTestOn [ "x86_64-linux" "aarch64-linux" ] ./oddjobd.nix {};
   oh-my-zsh = handleTest ./oh-my-zsh.nix {};
   ollama = handleTest ./ollama.nix {};
+  ollama-cuda = handleTestOn ["x86_64-linux" "aarch64-linux"] ./ollama-cuda.nix {};
+  ollama-rocm = handleTestOn ["x86_64-linux" "aarch64-linux"] ./ollama-rocm.nix {};
```

Comment:
question (optional): Would it be possible to use `runTest` instead ?

Reply:
What's the difference between `handleTest` and `runTest`? Does `runTest` use a vm too? Is `runTest` effective for testing nixos modules? Is there any documentation you could point me to (I couldn't find any)?

Reply:
`handleTest` and `runTest` are using the same technologies for testing. The only difference between the two is that writing tests using `runTest` seems to be simpler to write, a simple lambda function and it's done (there are examples in `nixos/tests/all-tests.nix`, check them out), while `handleTest` is a bit more complex with a call to `import ./make-test-python.nix`.